lib/transformations/occurenceOrderPlugin/__testfixtures__/occurenceOrderPlugin.output.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    plugins: []
```

Comment:
I was thinking about it but still not sure about it. Do you think it won't be too obscure to just remove it from the config?